src/extension.ts

```diff
@@ -156,7 +157,95 @@ export function activate(context: vscode.ExtensionContext) {
     }
   )
 
-  context.subscriptions.push(getAssetDetails, publishAsset)
+  let downloadAsset = vscode.commands.registerCommand(
+    'ocean-protocol.downloadAsset',
+    async (config: any, filePath: string, privateKey: string, assetDid: string) => {
+      vscode.window.showInformationMessage(`Private key: ${privateKey}`)
```

Comment:
Let's not show the private key, that feels weird

Reply:
ok, I'll delete it from publish too